packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -50,17 +50,9 @@ export function initialFetchSpaces(sparkInstance) {
 
 export function fetchSpaceAvatar(sparkInstance, space) {
   return (dispatch) => {
-    const currentUserId = sparkInstance.internal.device.userId;
     dispatch(fetchingAvatar(space.id));
     if (space.type === `direct`) {
-      // Support for direct messages with users who no longer exist
-      if (space.participants.length > 1) {
-        const otherUserId = space.participants.find((p) => p.id !== currentUserId).id;
-        return sparkInstance.people.get(otherUserId)
-          .then((person) => {
-            dispatch(storeAvatar(space.id, {url: person.avatar}));
-          });
-      }
+      return Promise.reject(new Error(`Direct spaces should use avatar service`));
```

Comment:
I'm going to create another tech debt ticket for that. I was looking into doing that with this PR but I was 100% on it.